src/timeline/TimelineInputs.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+
+class TimelineInputs extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            valid: true
+        };
+        this.handleMinBlur = this.handleMinBlur.bind(this);
+        this.handleMaxBlur = this.handleMaxBlur.bind(this);
+
+    }
+
+    handleMinBlur(e) {
+        console.log(e.target.value);
```

Comment:
i know you'll be getting rid of all these consoles eventually. Just pointing out there's a lot :)